gradle/libs.versions.toml

```diff
@@ -2,13 +2,14 @@
 compose = "1.3.0-alpha01"
 composecompiler = "1.2.0"
 coroutines = "1.6.0"
-espresso = "3.5.0-alpha07"
+espresso = "3.4.0"
```

Comment:
Can you run the `AboutScreenTest` and make sure it works first, before downgrading the `Espresso` version. I think there was a bug on `Roboelectric` that needed a more recent version of `Espresso`

Reply:
Got it, I will upgrade espresso version to the previous version. There was also a bug with dokka Version (or so I think :-) ) , the dokka documentation tasks kept failing on my end, I upgraded it to 1.7.10 (the most recent version) and its working now. I'll push the changes in a few.

Reply:
Done

Reply:
Thank you. Resolving this thread